.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+**Motivation**
+
+<!-- Why is this PR exists? What are the goals of the pull request? -->
+
+**Description**
+
+<!-- A clear and concise general description of the changes of this PR commits -->
+
+<!-- If applicable, add screenshots to help explain your solution -->
+
+<!-- Link to issues: Resolves #111, Resolves #222 -->
+
+**Steps to test or reproduce**
+
+<!--Steps to reproduce the behavior:
+```sh
+git checkout <feature_branch>
+lodestar beacon --new-flag option1
+```
+-->
```

Comment:
maybe add notice to add "closes #issue_number" if applicable?

Reply:
It's here 
https://github.com/ChainSafe/lodestar/pull/2195/files/6b6ed4d0c7072adcab7edf210470c01e2a200d52#diff-b2496e80299b8c3150b1944450bd81c622e04e13d15c411d291db0927d75fd6bR11
Let me know if I should phrase it more clearly

Reply:
Maybe add it as text rather than comment, so people are force to delete if not appropriate ? :)

I'm good both ways